xml/docker_installation.xml

```diff
@@ -102,69 +102,90 @@
     </para>
     <screen>&prompt.sudo;systemctl enable docker.service</screen>
     <para>
-     This will automatically enable docker.socket in consequence.
+     This will automatically enable <systemitem>docker.socket</systemitem>
+     in consequence.
     </para>
    </step>
    <step>
     <para>
-     In case you will use Portus and an SSL secured registry, open the
-     <filename>/etc/sysconfig/docker</filename> file. Search for the parameter
-     <parameter>DOCKER_OPTS</parameter> and add <literal>--insecure-registry
-     <replaceable>ADDRESS_OF_YOUR_REGISTRY</replaceable></literal>.
+     If you will<remark>toms 2019-02-04: will -> want to?</remark> use
+     Portus and an SSL secured registry:
     </para>
+    <substeps>
+     <step>
+      <para>Open the <filename>/etc/sysconfig/docker</filename> file. Search for
+       the parameter <parameter>DOCKER_OPTS</parameter> and add
+       <literal>--insecure-registry <replaceable>ADDRESS_OF_YOUR_REGISTRY</replaceable></literal>.
+      </para>
+     </step>
+     <step>
+      <para>
+       Add CA certificates to the directory
+       <filename>/etc/docker/certs.d/<replaceable>REGISTRY_ADDRESS</replaceable></filename>
```

Comment:
I would make this two separate steps:
Add the CA certificates to the directory...
Copy the CA certificates to your system...